src/swarmus_pioneer/launch/pioneer_bringup.launch

```diff
@@ -8,10 +8,31 @@
     <group ns="$(arg robot_name)">
         <!-- Roboclaw -->
         <node pkg="roboclaw" type="roboclaw_node" name="roboclaw_node">
-            <!-- TODO: set roboclaw config -->
             <param name="~serial_port"  value="/dev/roboclaw"/>
             <param name="~baudrate"     value="115200"/>
             <param name="~roboclaws"    value="1"/>
+
+            <param name="~enc_publish_frequency" value="100"/>
+        </node>
+
+        <node pkg="roboclaw" type="diffdrive_node" name="diffdrive" output="screen">
+            <param name="~roboclaws" value="1"/>
+            <param name="~steps_per_meter" value="76008.542519038410"/>
```

Comment:
76008.542519038410: seems oddly specific... How was this generated?

Reply:
Voodoo magic... Actually its the result of an equation I calculated outside the code: Ticks/mm = (EncodeurResolution*4*GearRatio)/(WheelDiameter*PI) = 76,00854251903849974901842759851 (the 4 is because its a quadrature encoder)